MasterServer/Program.cs

```diff
@@ -6,114 +6,146 @@
 
 namespace MasterServer
 {
-	class Program
+	internal static class Program
 	{
+		private static bool s_Daemon = false;
```

Comment:
Also maybe rename it to `isDaemon`? :thinking:

Reply:
Done